docs/pages/reference/troubleshooting/installation.md

```diff
@@ -20,6 +20,25 @@ Anytime a `mesheryctl system` command is executed, a series of preflight checks
 
     Situation: Unable to start Meshery Server with `make run-local` due to error of `key/value size is invalid`
 
+## Setting up Meshery using Kind or Minikube
+
+The difficulty with Minikube and Kind clusters is that they typically don't have a loadbalancer, whereas Meshery gives its users complete control over how networking is configured. As a result, networking for these deployments is more complex. Few of the alternatives are:
+1. Use the MetalLB minikube add-on that provides load balancing. `minikube addons enable metallb`
+
+    MetalLB setup: [link](https://kubebyexample.com/learning-paths/metallb/install)
+
+2. Use minikube tunnel to expose services. `minikube tunnel`. 
+
+    Docs: [link](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-tunnel)
+
+    A more simple way to resolve this issue can be `port-forwarding`. Run the following command in terminal:
+
+    `kubectl port-forward service/meshery 9081:9081 -n meshery`
+
+3. For `kind`, I prefer installing MetalLB with a custom configmap.
```

Comment:
Let's avoid first-person statements in Meshery Docs, using active voice instead; e.g. "You can install MetalLB with a custom ConfigMap".